### DISCUSSION TITLE
Midterms

### DISCUSSION THREAD

**Discussion by yohman** [yohman/21S-DH151#47] [Assignments]
Read the requirements for your midterm [here](https://github.com/yohman/21S-DH151/tree/main/Midterm%20and%20Finals).

Submit:
- Group name
- Group members
- Link to updated proposal
- Link to presentation (if applicable)
- Link to GitHub pages url of midterm project

**Comment by kimjee8955** [comment]
- Datart
- Annmarie Cong, Eustina Kim, Michelle Lee
- Updated Proposal: https://github.com/kimjee8955/DH151-Datart/blob/main/Group_Assignments/UpdatedProposal.md
- Presentation: https://docs.google.com/presentation/d/1uPfw9yJF-4B_fnn02htmEfAFqQg6xaQVLbRJqT9zAa4/edit?usp=sharing
- Website(open in Safari): https://kimjee8955.github.io/DH151-Datart/

**Reply by derekwang99** [reply]
I really love the idea behind your project! If it did end up as a mobile app, I would use it because I have definitely run into these issues before while exploring LA. I think one thing that the visualization could benefit from (and would apply to everyone's maps) is having the "Layers" tab stay open by default. I think that would reduce some initial confusion as to how to navigate the map. Additionally, I think an option to consider is changing the marker colors from red/blue to a gradient blue (e.g. earlier years will be a light blue and later years will be a darker blue) to attribute some meaning to the color.

**Reply by danielliuzy** [reply]
I like the features of the website, especially the Google street view incorporation with the location of the artworks! The website is also quite polished and the styling is quite consistent. I feel like an improvement to the user experience can be made by adding a Google street view button to the sidebar, so when the user clicks on a marker, they can click on the button in the sidebar to go to the street view instead of having to zoom in on the exact location of the artwork on the map, which can be a mild annoyance.

**Reply by JuliaWood1** [reply]
I was very impressed by your midterm project. I just drove to melrose today, and was very  surprised by all the murals around LA so I would definitely utilize this project when out and about in LA to learn more about the art around me. I loved the google street  view feature, that was  very impressive. I also loved the website logo. Something I thought your group could add would be like a "art on the way" functionality. So you could input your starting and ending location and then your map would show which art installations people  should see on the way. This might be hard to implement but it would be a very fun way to get people to visit your site frequently! Overall great work. 

Edit: I think your links to your website might be broken because I could not click on them.

**Reply by alexthegreat1** [reply]
I really like this project and I feel like it could see some consistent use because it isn't something that people visit once, digest, and then move on. They would visit it each time they wanted information about public art in LA county. Also love the site logo and the new randomize function! I also really like the visual design of the site overall. It looks very appealing and you can jump right in from when you visit the site.

A couple of notes:
- The Google Street View marker might be confusing for some users since it's always there from when they first visit the site. Maybe you could implement a feature where the marker doesn't show up until a user presses a certain button. This way they could toggle the marker on and off.
- I also think it would be really cool to implement the user's current location onto the map and show the nearest art markers to them if they wanted. However, I'm not sure how difficult this would be to implement in Leaflet.

**Comment by kaicolorado** [comment]
- cARTographers
- Alex Haddad and Kai Colorado
- Updated Proposal: https://github.com/kaicolorado/DH151-Group/blob/main/Group%20Assignments/Proposal.md
- Presentation: https://docs.google.com/presentation/d/1Gju0Mb9l1xlU4IFludYSLUpldXV6LtWk2K79OlbE1XA/edit?usp=sharing
- Website: https://kaicolorado.github.io/DH151-Group/index.html

**Reply by alicelu170** [reply]
I really like your project idea and the colors used to create an overlay design; I think having colors build up helps visualize how different states may be more likely to implement arts education policies. I have a few notes for possible improvements:

- Perhaps consider either providing a key for what certain color overlay combinations mean, or just use iterations of the same color and same opacity so that building up colors will make a state darker.
- Once standardized test scores are put in, I think it would be helpful to have a layer that combines arts education policies and test scores. That is, it would be helpful to show just how policies are affecting scores.
- If possible, binding a popup to states that are moused-over to show the policies that are visible on the map would also help in making the data more visually accessible!

**Reply by lisalou36** [reply]
I really love this topic guys, and you have clearly done so much work already -- great job! I like Alice's recommendations for additional components to clarify certain aspects. The website itself is very intuitive, it would just be more for the content, which I believe you mentioned in your presentation you will be working on towards the final. I think the only point of confusion for me was how the states change colors when there are more than two layers present; I am not sure how to read it beyond two. With those things in mind, I look forward to seeing the final product, great work!

**Reply by zonaliao** [reply]
I really like how you guys had an activated layers legend on the bottom that would increase each time a new layer was checked to appear on the map. The expandable sidebar on the left is cool too and I think maybe you guys can somehow incorporate a hide or expand function for the activated layers maybe because the list does get pretty long, but the fact that it's translucent makes it so that you can still see the map from behind which is nice. Another thing that I think is already mentioned above is that the colors start to blend with each other after checking so many of the options, so maybe having a certain way to differentiate which colors are on a state would be a good touch.

**Reply by hanarama** [reply]
I really love how the activated layers key that you included. With so many different layers, it's very helpful to see at a glance which ones are visible. I also really like the expandable sidebar on the right side. I think that if you're able to provide information on the different education policies in this area, it would allow the user to draw a conclusion on the effectiveness of a given policy (or combination of policies). I also agree with Alice that having a mouse-over affect would be really helpful. I would love to be able to mouse-over a state to see which policies are in effect there. This would be especially helpful when navigating the test score layers.

**Reply by ainsleighdouglas** [reply]
I think this subject is really important and often over looked in state policies/funding. I agree with a lot of the comments above, specifically on clarifying that the policies are on the map page (having a key of sorts). I really like your expandable side bar and it would be super useful in constructing this key. I have a feeling these are already in the works for your final so additional suggestions I have is creating a color hierarchy  on your page. Having a light blue icon for your expandable side bar makes it easy to miss when overlaid on this map. I would also suggest varying your type size/font/color of your buttons so they stand out more.

**Reply by ec028** [reply]
The different active layers and how they all play into each other is really important; I think it really strikes home the fact that standardization for education standards across all states is weak at best, and is really up to the discretion of each individual state. Given how arts education constantly suffers from a lack of funding, I think this topic is one that is very important to dive into. I think explaining the connection/correlation between having a layer for 4th grade and 8th grade on the map page might help in justifying why you chose those years in particular -- was this a particular standard the states decided on, or was it merely the only year where all states had the data? Overall, I really enjoyed playing around with the different settings and seeing where the overlap of standards were.

**Comment by alekhyavittalam** [comment]
- State-of-the-Climate
- Natasha Grace, Zona Liao and Alekhya Vittalam
- Updated Proposal: https://github.com/alekhyavittalam/state-of-the-climate/blob/main/UpdatedProposal.md
- Presentation: https://docs.google.com/presentation/d/1vJI6KaQS6iuJIr4hdw6RsmiESZR35APG3He8JVSB6ok/edit?usp=sharing
- Website: https://alekhyavittalam.github.io/state-of-the-climate/Midterm/index.html

**Reply by yjwsb233** [reply]
Your website and project are really important! I like how users can see the status of each country. Would it be possible to create a choropleth map that shows the status of each country through color gradients? I think that would be really cool! Also, I noticed that your base map is in the language of each country. For the sake of consistency, maybe add a basemap that's all in English? Can't wait to see your final result!

**Reply by michelle14lee** [reply]
I like the organization and coloring of your map, and the issues you highlight are worth bringing attention to. Since your topic is about death rates, I feel like the green coloring is a bit misleading (because green is often associated with good, positive things). Perhaps using a different color scheme or even using icons to represent high/low death rates would make more sense (for example, an arrow pointing up for increasing death rate, pointing down for decreasing).

**Reply by jomh17** [reply]
I think that you should consider representing your data points in a spot gradient based on death rates. So low death rates would be pale and high death rates would be saturated. I think it could convey the data in a more impactful way so that the user can understand the data without having to click on a point. I think it could also be cool to use data about the pollution levels in each place in polygon so the death rates could lay on top of that layer. It would be really interesting to see how pollution levels correlate with death rates. For example US could have a higher pollution rate but low death rate, etc. Adding pictures of pollution in the about page would also be a really effective way to get your point across.

**Reply by jmtanenbaum** [reply]
I really love the idea of this map. I think the coloring of the data is useful, but you need more text and narrative. What does low income countries mean? Why is this the case? What are the historical factors? Data needs context.

**Reply by ryanorlando** [reply]
I really like the way your project is looking! I feel like I learned a lot moving around on your map, which is one of the most important aspects of this project. I think one way you could heighten the impact of this site is to include an area that tells people how they can help with this issue. How can we take small steps to reduce our own environmental impact, and potentially indirectly save lives. I know that many of these issues are far broader than just individual contributions, so maybe an addition of instructions or advice on how to get involved lobbying for better climate protections. This is an important issue, and your project can help people understand it better!

**Reply by JuliaWood1** [reply]
Hi State-of-the-Climate team! I think this project is a really important issue to highlight. I really like the color and font choices of the text on the screen as well as the image in the bottom left hand corner. I agree with what people said above about perhaps using a color gradient to more clearly show the pollution percentages. I also think it would be really interesting to look how pollution deaths are correlated with other geographical/political factors like perhaps the country's GDP or population or health care system.

**Comment by frida-barragan** [comment]
- The Health Mappers 

- Frida Barragan & Kate McInerny

- Updated Proposal : https://github.com/frida-barragan/DH151_Group/blob/main/MidtermProposal.md

- Presentation https://docs.google.com/presentation/d/1xnqlLzOxQjcZvmLTWbd6GFSU9sg5aLXFYyXpovh7Rp4/edit?usp=sharing: 

- Website : https://frida-barragan.github.io/DH151/Midterm/Frida/index.html

**Reply by michelle14lee** [reply]
Mental health is always a really important issue of discussion, and I like how you have a landing page to introduce the site! I think it's not very obvious navigating between the maproom and other pages currently. If you could move the hamburger menu symbol to the header and make the logo at the top left corner redirect to the home page that would help make the UX more intuitive! I think it'd also be neat if the "Service Area" items on the sidebar could disappear as map layers were unchecked to make the map more dynamic.

**Reply by alicelu170** [reply]
I really like the landing page as well and I thought the hidden navigation bar was a nice touch! I also appreciated the practical uses of your project, as users can potentially refer to it as a resource. 
One component that I think would make the map more visually intuitive is to color-code the polygons you use to delineate each service area. Additionally, I think having the items in your sidebar be clickable, such as zooming in to the location of the provider, would greatly enhance the interactivity of your interface.

**Reply by jomh17** [reply]
Okay firstly your website is super cute, and the topic is one that is really important. I would maybe make the borders around the buttons a little thicker and the font larger so it is more visible. Maybe make the sidebar the same color as the homepage so it is more cohesive when you click on other pages. I can see that you guys are trying to keep a muted color scheme but I think that the points and polygons on the map could be brighter and the map icons larger. It would also be super cool to have a zipcode or address space that a user could input the zooms into the map where their location is. Adding the address of each place in the popup could be valuable information for the user so they don't need to go to another site to look it up. Having different colored polygons for each service area could also be helpful.

**Reply by MaryJaneHowland** [reply]
Your use of animated gifs made your site more engaging, which I really liked. I also really liked how you incorporated infographics about mental health into your site, as it made it more informative and more visually engaging than if you were to just write our the stats/info. One thing I would think about changing is making the color of your text on your main page brighter/bolder, as it is a little difficult to read. Something else you may want to consider adding is a "resources" page that has different information/resources/links related to anybody who may want mental health services.

**Reply by jmtanenbaum** [reply]
This project seems essentially complete to me. I love the infographics to discuss the importance of income inequality in mental health issues. The GIFs and user interface are also really impressive. The only thing I can see being improved is adding some of the data about the areas themselves to the map.

**Reply by hoz-map** [reply]
Your project was inspiring and informative. I think this would be a very helpful resource for many people who are in need of mental health care. Additionally, I really like the mild color theme of your website. This is an important design component because some people who are having mental health issues can be sensory sensitive, which is why pastel or mild color is preferable. In terms of something you can add, you could add a direct link to the website on each pop-up on the map, which might make it easier for users to find more detailed information about the resource.

**Comment by LynnT2** [comment]
Fantastic Four
Labiba Alam, Haiqi, Adriana Romero, Lynn Tieu
Updated Proposal: https://github.com/LynnT2/FantasticFour/blob/main/Midterm/updatedproposal.md
Presentation: https://docs.google.com/presentation/d/10Nuee1YaoXfA8vkx6ApehIzqA01WqUKmP-a9GIpFDDg/edit?usp=sharing
Website: https://lynnt2.github.io/FantasticFour/Midterm/index.html

**Reply by kimjee8955** [reply]
I absolutely loved the design of the website! My favorites were the animation on the landing page and the timeline. I think an interesting idea to implement for the final is letting the users put their stories on the map, here's an example I found that was done by the Daily Bruin when they reported on sexual assault on campus: https://features.dailybruin.com/2019/beyondthereports/

**Reply by alekhyavittalam** [reply]
I love your website and the animations used! I especially liked the clusters and the timeline. I think it would be a good idea to have a default view button since after one zooms into the clusters, they will have to zoom out to go back to default view.

**Reply by LynnT2** [reply]
@kimjee8955 Omg we were actually thinking about having a story feature like that but weren't sure how to implement it. Thanks so much for the reference!

**Reply by LynnT2** [reply]
@alekhyavittalam We're definitely going to add a default button once we figure out the code haha, thanks for the feedback!

**Reply by zonaliao** [reply]
I really liked how you guys created a fade-in homepage with that first prominent fact before the viewer can click on the "Delve Deeper" arrow because it makes the introduction to the topic more impactful. The color-coded markers with the numbers on them and the timeline points are really unique, and I like how the overall website style is very clean and easy to understand. It might be cool to add a feature where if you click on a certain dot on the timeline you can click on a back or next arrow to continue to the next event on the timeline so that the viewers won't have to keep clicking on the exit button to go back to the timeline before clicking to view the next information point. It might make the timeline flow better too so that someone using it can see all the events in order with more ease just by clicking "next."

**Reply by natgrace** [reply]
Wow I hit my required two comments but I can't not leave a comment here!!! This map was so well done, color scheme showed how serious the topic was and the site really captured how important getting this information out was to you guys. I took an Asian American studies class in the Fall and although I am saddened by how many incidents I've yet to learn about, I feel like I was able to use what I was taught to really resonate with the information your site. I don't really have any suggestions for improvement but I would love to learn a thing or two about the coding for the timeline! It's such a useful part of the website/data and something that's hard to capture as well as you guys did in your map! Great job!!

**Comment by derekwang99** [comment]
Ocean Pollution Posse
Julia Wood, Eli Aranita, Derek Wang
Updated Proposal: https://github.com/eliaranita1/project1/blob/main/Midterm/UpdatedProposal.md
Website: https://eliaranita1.github.io/project1/landing.html

**Reply by lisa58** [reply]
The title page is so cool! It really engages anyone visiting the site right away. I think it would be neat if the different buttons end up leading to different part of the site depending on your answer instead of the home page every time.

**Reply by yjwsb233** [reply]
I love the intro page to your website! (Would it be possible to let the users go back to the questions to click on other answers?) Your topic is so important and it's something I care about a lot. I know that there are a lot of films on similar topics like Seaspiracy – do you think it would be a good idea to include them on the resource page? I think it is very cool to have so many layers of data on your map, but the ones with 360k points are taking some time to load. Is there a way to convey the path of ships without using all the points?

**Reply by ryanorlando** [reply]
This is a great start to your project! All the data looks good, especially the way you presented in in Tableau. I also like how you are including resources for further education. I can tell that this is a topic that you all are passionate about and that is conveyed through the details of your site. One suggestion I would make is to find some real life images of ocean pollution beyond just stock photos, I think it would really increase the impact of your information. I know once the full map is up and running it will provide a good glimpse into how widespread this issue is, but with such an important topic, any additions to promote urgency will be super helpful. Super excited to see how this turns out!

**Reply by anncong** [reply]
I really like how different your project is and it looks promising to go beyond the class. Incorporating the Tableau data was definitely a nice touch to your overall project.  I noticed you guys added drop down menu-items, which is pretty cool for faster-navigation throughout your site :). Also I like how you guys incorporated the hover CSS states! I am excited to see how your site will progress. Some possible things you guys can add/improve on your site are:

- Making sure the color (background and text) have good contrast 
- Usually left-aligning large chunk of texts like paragraphs will help users navigate the readings easier
- the texts on the bottom on the fishing vessel page seems very "packed", so you guys can maybe add margin spacing to make it less cramped up.

**Reply by hanarama** [reply]
Your website is incredibly engaging and informative. Even with some pages unfinished, your intent is clear. You're planning to include a lot of really crucial information to understanding this issue and how to respond to it. I can't wait to see the finished product. I think that it will be really important that you include contextual information for the map itself. Beyond what they layers tell us, there's little way to really interpret the information displayed. You may want to explain the significance of the dots, how much pollution they represent, and what the ship pathways are. I also agree with anncong that making sure text has enough space and is easy/pleasant to read will really improve the UX of your site.

**Reply by sebdills** [reply]
This is super cool. You used the data very effectively and proved correlations/causations well. I would suggest you make all graphs you made in Tableau more concise. There are currently tons of them and I feel as if it kind of confusing to read. Otherwise everything was cool, I liked how you asked the user a question at the beginning of the sites wire flow.

**Comment by MaryJaneHowland** [comment]
Group: Happy Tweet
Member: Ivan Ma, Josephine Meyer-Hogan, MJ Howland
Updated Proposal: https://github.com/ivanma9/HappyTweet/blob/main/Group%20Assignments/UpdatedProjectProposal.md
Presentation: https://docs.google.com/presentation/d/1rBduSiwmDcYgFQQv6lMZ3ypZsUadjF_N9IzYcco5d_w/edit?usp=sharing
Website: https://ivanma9.github.io/HappyTweet/index.html

**Reply by lisa58** [reply]
Love the pictures on the icons and the old Twitter aesthetic! I also feel very inspired by the idea of having a button that resets the layers on the map. I think it would be a good idea to have a timeline correlating the amount of screen time spend on social media with happiness levels if that's possible.

**Reply by LynnT2** [reply]
Love the design simplicity, it was easy to figure out how the map works thanks to those icons. It would be cool to see more emojis being used by having them represent the happiness level per marker on the map.

**Reply by danielliuzy** [reply]
I love the idea of taking data from Twitter, as it is definitely very interesting but can be quite challenging to find suitable geo-based data. I think a cool feature for the website would be adding popups to the markers to look at more information of the tweet, or opening Twitter on a new tab to view the corresponding tweet when clicked on it.

**Reply by natgrace** [reply]
I love the twitter design, it really ties in the map and the purpose behind your data! I've been an avid twitter user for over 10 years so this topic is super interesting to me and I'm positive I have A LOT to learn from it. I'm excited to see what other features you guys are planning to add! My one suggestion would just be to add popups to the markers to see more about the happiness levels, perhaps a "likes" or "retweets" count (like on a tweet) to show how happy they are? Wonderful work, I really enjoyed this!

**Reply by kaicolorado** [reply]
I like your website's design and ability to quickly switch between maps. I think it would be a lot more informative if the points were able to expand to give extra information on the happiness or twitter metrics associated with the points. By adding a hover or on-click popup for the points, the user would be able to more easily see the association between happiness and twitter usage. Also, when the webpage first loads, the map is initialized with Africa in the middle of the map and then shifts to the United States when a filter is activated. I think it would be less jarring for the user if the map was initialized at the same view that it has for the filters.

**Reply by hoz-map** [reply]
I love how you designed your websites, such as using the hashtag for a button and Twitter-themed color! One suggestion would be an adding a short description of what both Happiness Map and Twitter Map indicate and how the data are collected. Overall, I really enjoyed your websites!

**Reply by kaisanwatanabe** [reply]
I really appreciate your intuitive webs design, and also the decision to implement twitter data! I think being able to see data from social media platforms is exciting because it's a much broader audience than a controlled dataset. Being able to see the happiness map and twitter map at the same time makes it easy to see the correlation between the both of them too. I think it would be helpful to have short excerpts about what the twitter and happiness data means, and perhaps some further information per popup. Nevertheless, 'm excited to see what additions you guys bring to the websites!

**Comment by lisalou36** [comment]
Group: Environmental Racism in Carceral California
Members: Hannah Lien, Julia Tanenbaum, Lisa Kahn
Updated Proposal: https://github.com/jmtanenbaum/DHEnviroIncarceration/blob/main/Updated%20Project%20Proposal.md
Website: https://jmtanenbaum.github.io/DHEnviroIncarceration/index.html

**Reply by kimjee8955** [reply]
I loved the use of choropleth layer and how the map effectively brought environment and incarceration together. It was especially interesting to see that a lot of the prisons are located in places with most pollution! I think it would be a good idea to add a legend that shows quantitative information of the colors! (If the waste layers have quantitative information).

**Reply by adrianaromero819** [reply]
As someone who is interested in prison abolition, I really enjoyed the presentation and this website! It emphasizes two important issues, mass incarceration and environmental hazards, and demonstrates the relevant relationship between the two. Environmental racism takes various forms and it was interesting to visually view it on a map framed in relation to incarceration in California. The featured stories are important and help personalize the realities behind the map. The choropleth map is really impressive and the layers do a great job at capturing the different threats. I think having a legend that shows the color-coding of the map and includes information about the markers (ex. Black dot = prison location, name, prison population) will make the interactive experience easier and simpler! I also enjoyed the information on the sidebar, bolding or coloring certain text that you think is important to the site (ex. “police, pollution, and prisons”) might emphasize the sense of urgency on this topic and help the narrative flow.

**Reply by kmcinerny** [reply]
I love how you start your maproom’s written narrative with a quote that expands how we think about “environmental threats.” Prisons and police of course shape environments and social determinants of health. As you point out, prisons also contribute to destruction of our physical environment/climate. I think your map is beautiful and works well with all its layers in that it visually highlights pollution surrounding jails & prisons. At first glance I thought the red/orange areas were polluted, while blue/ green areas were not. Now I understand that those two colors represent different types of pollution. I wonder if there’s a way to make that a little bit clearer, perhaps by choosing solely “warm” colors? At the same time, though, I understand choosing a range of colors so that there is visual distinction. So I don’t know if this is helpful. But I also think you could clarify what you mean by “the darker the color, the more polluted the area.” Does this have to do with layering? Are the blue or red areas more polluted?

**Reply by kaicolorado** [reply]
I really like your website and think that the topic that you're discussing is a great intersectional analysis of the dangers of the United States prison system. I think that your sidebar could possibly be broken up with some headings or divisions in text to make the information easier to read. The text in the sidebar that describes how to read the map might be better formatted as a legend overlay on the map. This could be done by showing the ranges of the three colors that show levels of pollution. Possibly you could link to the featured stories on the map or denote them with a special symbol to highlight those stories.

**Reply by ainsleighdouglas** [reply]
I think you guys have a highlighted a very critical subject in a very successful way. I think you choropleth map does a great job of showing the overlap of the the four layers you have created. The only factor that I would maybe rethink is digestibility. Your group has a lot of content to cover and information to share but upon landing on your home/map page I encounter a dense text and map. This subject would require time to fully understand no matter what but I suggest categorizing the information further into smaller pieces of the project.

**Reply by ivanma9** [reply]
I really like the nav bar and the logo and typeface in that section of the site. In the map part of the site, the font of the text is back to Times New Roman and it looks kind of old, and it did not make me want to read it. I would say to change the typeface of that portion of the site and maybe think about the amount of information you put on the sidebar. Also can put a heading for how to use the map as they all fell in the same area of text of your mission statement. I was not sure what the search bar was for, maybe a little title or predefined text inside the bar as instructions on what to type in the bar to properly use it. I typed in a city in California and nothing happened? 
The map looks beautiful with all the layers happening on the map and the option to remove or add the layer back in. Very smart move in terms of filtering and visualizing the data. I think you mentioned this in your presentation, but the order of what select and deselect change the look of the map. I think a possible solution is to store the "layers" in a fixed array, and then add the actual layer if it is not null (if selected) in the actual map, so you can have a consistent ordering for the layering. I am not sure if it was just my internet, but the map was kind of lagging for me. 
I like the idea about the stories under the map. Honestly, in my user experience, I was kind of caught up in the scroll bar of the map, and it took me to figure out that there was another scroll bar to get to the nav bar and stories section. Maybe add some visual pointers that could lead users to make that transition smoothly.
This might be more political, but I was unsure on how the map shows the racial injustice of pollution and environmental factors. I guess the map was kind of confusing in the alignment with your topic in that I was not sure what is the "cause and effect" and how I was suppose to see that in the map.

**Comment by hoz-map** [comment]
Group name: Rethink Recycle
Group members: Alice Lu, Emily Chau, Hozumi Oshida
Link to updated proposal: https://github.com/hoz-map/Rethink-Recycling/blob/main/UpdatedProposal.md
Link to GitHub pages url of midterm project: https://hoz-map.github.io/Rethink-Recycling/Midterm/index.html

**Reply by alicelu170** [reply]
https://hoz-map.github.io/Rethink-Recycling/Midterm/Week%205.about%20page/index.html

**Reply by alekhyavittalam** [reply]
I really liked your map and how you incorporated a color code as well as the size of the circle to show the intensity of the situation. If possible it may be cool to show the directionality of the exports and imports between countries.

**Reply by adrianaromero819** [reply]
We always talk about the importance of recycling but we rarely discuss where plastic waste is imported and exported. Starting with the world map allows the user to visualize the fact that recycling alone does not solve the problem of plastic waste, and literally places the narrative of global waste networks “on the map.” I really like the use of the sidebar to fly to the different locations on the map and using the colors/circles to differentiate imports and exports. Grouping numerous countries under a sidebar button can reduce the amount of scrolling to find a specific country and help with user interactivity. For example, condensing the side-bar buttons to represent continents (or however you want to group areas) and then having individual countries drop down when the button is pressed.

**Reply by justisg** [reply]
I love your color scheme and how it relates to your topic! I also think the design of the markers makes a really clear and strong point connecting to your project. I think the relevance of your topic and its clarity definitely makes it extremely impactful. I definitely agree with Adriana about grouping the countries together to make it easily navigable. I also think your site could make use of a page that is a more in-depth description and can maybe make conclusions on what your map is showing.

**Reply by alexthegreat1** [reply]
I really like your site's visual design and the map design itself! The way your markers are displayed is a really cool way of visualizing plastic waste imports/exports that I hadn't actually seen before! I also really like the idea of your site. The data you're mapping out really helps people understand the scale of the US plastics import/export trade and how wide it branches out.

Some notes:
- I think a really neat addition to the map would be having arrows point to and from countries for imports and exports. To add to that, maybe you could have the corresponding circles and the arrow itself get bolder or thicker as the user hovers over it so that people have an easy way of seeing which circles are connected (since the import/export circles seem to be paired a lot of the time).
- I also think a nice fix would be making it so that smaller circles aren't overlapped by larger circles, since sometimes if I try to get info on a smaller circle, only the larger circle's data shows. I don't know how difficult this would be though.
- One minor thing is I think separating your kg numbers with commas would help users understand the info faster.

**Reply by ivanma9** [reply]
I really liked the colors going on with the design. I think the green and white as positive colors and red as negative really goes with the theme of the recycling. The circles are really cool indications for markers as it really represents the density of the waste. 
I think that some things that you guys could think about:
There are many tabs on the sidebar, which are not very useful in that there are so many and the name of the button is kind of vague like "world". Maybe play around with the typeface of the markers or maybe display that fun fact of information in your marker somewhere else. I was kind of confused on the red and green concentric circles in Kansas. Maybe clear up the meaning behind that marker somewhere. It also says click on partner country to zoom in, but there are multiple entries of the same countries, so maybe group the markers with the same country as the same point since it is by country.

**Comment by ryanorlando** [comment]
Group name: Team Data
Group Members: Ryan Orlando, Ainsleigh Douglas, Sebastien Dills
Updated Proposal: https://github.com/ryanorlando/DH151-Project/blob/main/UpdatedProposal.md 
Website: https://ainsleighdouglas.github.io/DH-151/Midterm2/index.html

**Reply by derekwang99** [reply]
I really like your guys' topic! Very relevant to the current state of society. I think one suggestion that I have is that the "Map" page could retain the navigation bar so the users don't have to manually click the Back button to navigate back to the home page. I think this would improve the consistency across all pages as your other pages keep the nav bar.

**Reply by LynnT2** [reply]
I loved the clock widget idea that you mentioned during your presentation, it sounds like a really effective way to get your users to pay attention to the issues you're bringing up about these data centers. I agree with Derek about having the map page having the navigation bar. I'd also suggest keeping a consistent theme for the entire site. The map page is aesthetically very different from the About and Introduction pages—I thought I was on a different site for a second.

**Reply by justisg** [reply]
I found your topic extremely relevant and the way in which you explained it both in your presentation and on the website was very clear and easy to understand. Your map and about page are really well designed. I definitely agree with the posts above on some suggestions, mainly keeping the color scheme/fonts consistent from the map to the other pages. I also really like the placement of your sources as it is very clear and visible, as that is a very significant part of your project.

**Reply by kmcinerny** [reply]
I'm interested to see how your site develops since this is a relatively new topic to me (though I have heard, for example, about how much pollution comes from "junk" emails). As you all mentioned in your presentation, breaking up the text with images and other media would make it especially digestible. You could maybe even bold or visually strengthen some of the most important takeaways from the written narrative. I like the aesthetics of your map and how the size of the bubble corresponds with number, while the shade corresponds with layer/category. Perhaps you could supplement the map with more analysis of what we're seeing. For example, I wonder why the countries that have a lot of data centers tend to have relatively less carbon emissions than countries without data centers. Moreover, is there research on why there are fewer data centers in continents such as Africa, South America, and other parts of the Global South? Already I think this map, with its green circles, has the potential to subvert some colonial conceptions of environmentalism.

**Reply by frida-barragan** [reply]
Your project is truly interesting because it is about a topic I don't have much knowledge in and your group presentation was really well-structured and it helped me understand more about the importance of data weight. I also think it might be valuable to provide some background information about the placement of these data centers in specific countries as some other people have mentioned. I think it might also be useful to relate the weight of emissions to research that highlights future consequences, etc. Overall, I think your map is informative and really "paints" a clear picture of what you are t4rying to say to the users.

**Reply by ec028** [reply]
I really appreciated the clean/accessible layout of your website. I appreciate that you've set out a section meant for research, so I would definitely like to see how that becomes your write-up and what sources you drew your analysis from in the future. I think it would be nice to see more of a connection between the two layers you included in your map; you mentioned that your research will provide the bulk of the narrative, but I think explaining the connections between the two factors and how that may lead to your conclusions will help integrate the map better into the site.

**Reply by kaisanwatanabe** [reply]
I really appreciate your group's decision to cover this topic. With increased usage of blockchain technology, it can be easy to forget how much ecological impact our reliance on this topic can have. Mapping electrical usage and CO2 Output through this map helps us to visually understand where and how much exhaust is created on the environment. I was really surprised to see such a large data center in Curacao, and am excited to read about your data analysis. The site currently runs rather slow for me (although it could just be my problem), it may be beneficial to look into ways of sorting data centers so that the map is easier to scroll through and zoom in on. Overall awesome project!

**Comment by justisg** [comment]
Group Name: The Musical Maps
Group Members: Justis Guardia, Daniel Liu, Kai Watanabe
Updated Proposal: https://justisg.github.io/TheMusicalMaps/UpdatedProposal
Presentation: https://docs.google.com/presentation/d/1K5Fg5_8NMOAAsgwoJGbELNIhacvFwMk3iOmauq0irwk/edit?usp=sharing
Website: https://justisg.github.io/TheMusicalMaps/Website/index.html

**Reply by lisalou36** [reply]
This map already looks so complete, great job guys! I'm very impressed with the search bar functionality as well. There is clearly a ton of data that you had to work with to get to this stage, so great job parsing it and displaying it in such a visually pleasing manner. In terms of recommendations, I'd be interested in seeing some data visualization charts on your conclusion page that you mentioned (i.e., trends of growth in genres). I think it might also be helpful to also have a section walking though the thought process for why your scope (genres & global versus the U.S.). Wonderful work!

**Reply by MaryJaneHowland** [reply]
I liked how your team included a page on your site specific to your data, as I think it is really important to show the user where all of the information is coming from. I also liked your logo a lot - it made your site more unique! One of the suggestions I have for your team is to shorten or make more concise the section explaining how to use your maps, as it was a little difficult for me to digest. Also, I was not sure how to go back to that main how to use the maps page after clicking on other pages, so you may also want to look into that further.

**Reply by anncong** [reply]
I really like the interface of your site, it was easy to navigate and the popularity bar corresponding with the size of the bubble is a cool feature you guys have. I also like the search functionality you guys included! I think some tiny details you guys can add in additional to your already well-made site are hover CSS states, and maybe a border to the buttons of a slightly darker color because of color accessibility (separating the button from the background more).

**Reply by natgrace** [reply]
I love your site! It's so fun to interact with and really informative. I thought the search bar was a useful addition to help expand how specific a user can be when looking for data and I had a lot of fun searching up my favorite artists. It was easy to navigate that I didn't even realize how long I was on it! One suggestion I would have is to make the first page a little less wordy (because I thought it was already simple enough to navigate without a lot of instructions!) but other than that, this is one of my favorite maps so far! Great job!!

**Reply by sebdills** [reply]
I like the idea of having a page explaining how to use the map but I would make it as bare bones as possible in terms of verbiage. I think you should be able to figure more stuff out at a glance. I do love this idea tho. The map idea itself and the data sets used are really really interesting. I would investigate as to why there is so little information on Africa. Personally I listen to a lot of French and English language North African rap so I know it exists and is out there. Look into that.

**Reply by frida-barragan** [reply]
Your project map and concept is really interesting! The map has so much information that I think was formatted really clearly so that the user could understand and interact with it. I think the popularity bar was a great addition to each marker but it might be useful to add the actual percentage (a metric for popularity) next to it so that it is a bit easier for the user to grasp the differences between the concert popularity.
